ban/db/fields.py

```diff
@@ -151,6 +151,21 @@ def coerce(self, value):
         return value
 
 
+class FantoirField(CharField):
+
+    def __init__(self, *args, **kwargs):
+        kwargs['max_length'] = 10
```

Comment:
Erf, we have an issue here :(
I see why you done that, but this will also impact the max length of the DB column. And we want it to be 9, not 10.